packages/manager/modules/pci/src/projects/project/serving/serving.routing.js

```diff
@@ -7,12 +7,9 @@ export default /* @ngInject */ ($stateProvider) => {
       transition
         .injector()
         .get('$q')
-        .all([
-          transition.injector().getAsync('lab'),
-          transition.injector().getAsync('namespaces'),
-        ])
-        .then(([lab, namespaces]) => {
-          if (namespaces.length === 0 || lab.isOpen()) {
+        .all([transition.injector().getAsync('namespaces')])
```

Comment:
Same here, I guess you can remove `.all`call

Reply:
fixed